src/components/tile/_tile.scss

```diff
@@ -0,0 +1,152 @@
+//-----------------------------
+// Tiles
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/layer';
+@import '../../globals/scss/typography';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/css--reset';
+@import '../../globals/scss/css--typography';
+
+@include exports('tile') {
+  .bx--tile {
+    @include layer('raised');
+    min-width: 8rem;
+    min-height: 4rem;
+    background-color: $ui-01;
+    border: 1px solid $ui-03;
+    position: relative;
+    padding: 1rem;
+
+    &:focus {
+      @include focus-outline('border');
+    }
+  }
+
+  .bx--tile--clickable,
+  .bx--tile--selectable,
+  .bx--tile--expandable {
+    transition: $transition--base $bx--standard-easing;
+    cursor: pointer;
+
+    &:hover {
+      border: 1px solid $ui-04;
+    }
+
+    &:hover,
+    &:focus {
+      .bx--tile__checkmark {
+        opacity: 1;
+      }
+
+      .bx--tile__chevron svg {
+        fill: $brand-01;
+      }
+    }
+  }
+
+  .bx--tile--clickable,
+  .bx--tile--expandable {
+    &:focus {
+      @include focus-outline('border');
+    }
+  }
+
+  .bx--tile--is-clicked {
+    @include layer('flat');
+    border: 1px solid $ui-04;
+  }
+
+  .bx--tile__checkmark,
+  .bx--tile__chevron {
```

Comment:
![image](https://user-images.githubusercontent.com/4185382/30129605-1057bfb4-930c-11e7-981f-d51641b65879.png)

Should add focus state: 

```scss
&:focus {
  @include focus-outline;
}
```